migration/sql-files/055-space-templates.sql

```diff
@@ -0,0 +1,27 @@
+CREATE EXTENSION IF NOT EXISTS "ltree";
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE space_templates (
+    id uuid PRIMARY KEY DEFAULT uuid_generate_v4() NOT NULL,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone,
+    deleted_at timestamp with time zone,
+    version integer DEFAULT 0 NOT NULL,
+    name text UNIQUE NOT NULL,
+    description text,
+    template text NOT NULL
+);
+
+-- Create a default empty space template
+-- @TODO(kwk): This can probably get a better name later.
+INSERT INTO space_templates (id, name, description, template) VALUES(
+    'b5beea58-fa52-43b1-a21d-dd6a8d6e61bb',
+    'Empty Space Template',
+    'This template just exists in order to map already existing spaces to a space template',
+    'work_item_types:'
+);
+
+-- Add foreign key to spaces relation
+ALTER TABLE spaces ADD COLUMN space_template_id uuid REFERENCES space_templates(id) ON DELETE CASCADE;
```

Comment:
I did it because the foreign table is also called `space_templates` and within there's an `id` field, hence `space_template_id`.